.github/workflows/ci.yml

```diff
@@ -61,25 +61,31 @@ jobs:
           args: --verbose --release
 
   no-std:
-    name: Check no-std compatibility
+    name: Check no-std target ${{ matrix.target }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - thumbv6m-none-eabi
+          - wasm32-unknown-unknown
+          - wasm32-wasi
 
     steps:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: 1.44.0
           override: true
-      - uses: actions-rs/install@v0.1
+      - run: rustup target add ${{ matrix.target }}
+      - name: cargo fetch
+        uses: actions-rs/cargo@v1
         with:
-          crate: cargo-nono
-          version: latest
-          use-tool-cache: true
-      - name: Check for no-std compatibility
+          command: fetch
+      - name: Build
         uses: actions-rs/cargo@v1
         with:
-          command: nono
-          args: check --no-default-features --features groups,pairings
+          command: build
+          args: --verbose --target ${{ matrix.target }} --no-default-features --features groups,pairings
```

Comment:
I switched back to testing general no-std compatibility with the `thumbv6m-none-eabi` target to reduce duplication, as I also want to explicitly test the `wasm32` targets.